NOTICE.txt

```diff
@@ -4494,7 +4494,7 @@ SOFTWARE.
 
 This product contains 'archiver' by Matthew Holt
 
-A library to handle direferen archive files (zip, rar, tar.gz...)
+A library to handle different archive files (zip, rar, tar.gz...)
```

Comment:
@amyblais - Is this line written by us or copied from somewhere else?

Reply:
@jespino added it in #15665

Reply:
Ah thanks for the blame analysis @jsoref. Good to be changed. :+1: